SocialFilter/sections/general_elemhide.txt

```diff
@@ -87,6 +87,7 @@ jp##.box-sns
 ~twitter.com##a[href^="https://www.twitter.com/intent/tweet?"]
 ###share_plz
 ##.sns-share
+##.article-header__sns-share
```

Comment:
If you add a generic rule, you have to provide strong rationale (no false positive and there are too many cases to address specific rules). Otherwise better `autograph.ismedia.jp##.article-header__sns` in specific.txt.

Reply:
Thank you. Fixed!